test/index.ts

```diff
@@ -83,6 +84,7 @@ function createWorkspace(): Blockly.WorkspaceSvg {
     renderer,
   };
   const blocklyDiv = document.getElementById('blocklyDiv')!;
+  ToolboxMonkeyPatch.install();
```

Comment:
Oh, I didn't mean to move this all the way up into `test/index.ts`. Instead, call `install` from `NavigationController.init()` and `uninstall` from `NavigationController.dispose()`

Reply:
However, I think that I'm distracting from the point of this PR by focusing on where that code goes, and it's fine to merge as is.

Reply:
Reverted the logic back to the way it was before since installation is messy here, and the monkey patches can go away soon (once v12 is available).